doc/sphinx/src/models.rst

```diff
@@ -3,3 +3,409 @@
+
+where quantities are defined in the :ref:`nomenclature <Nomenclature>` section.
+
+The settable parameters are the Gruneisen parameter and specific heat capacity.
+Although this differs from the traditional representation of the ideal gas law
+as :math:`P\hat{V} = RT`, the forms are equivalent by recognizing that
+:math:`\Gamma = \frac{R}{\tilde{C_\mathrm{V}}}` where :math:`R` is the ideal gas
+constant and :math:`\tilde{C_\mathrm{V}}` is the *molar* heat capacity,
+relatable to the *specific* heat capacity through the molecular weight of the
+gas. Since :math:`\tilde{C_\mathrm{V}} = \frac{5}{2} R` for a diatomic ideal
+gas, the corresponding Gruneisen parameter should be 0.4.
+
+Gruneisen EOS
+`````````````
+
+One of the most commonly-used EOS to represent solids is the Steinberg variation
+of the Mie-Gruneisen EOS, often just shortened to "Gruneisen" EOS. This EOS
+uses the Hugoniot as the reference curve and thus is extremly powerful because
+the basic shock response of a material can be modeled using minimal parameters.
+
+The pressure follows the traditional Mie-Gruneisen form,
+
+.. math::
+
+    P(\rho, E) = P_H(\rho) + \rho\Gamma(\rho) \left(E - E_H(\rho) \right),
+
+Here the subscript :math:`H` is a reminder that the reference curve is a
+Hugoniot. Other quantities are defined in the :ref:`nomenclature <Nomenclature>`
+section.
+
+The above is an incomplete equation of state because it only relates the
+pressure to the density and energy, the minimum required in a solution to the
+Euler equations. To complete the EOS and determine the temperature, a constant
+heat capacity is assumed so that
+
+.. math::
+
+    T(\rho, E) = \frac{E}{C_V} + T_0
+
+The user should note that this implies that :math:`E=0` at the reference
+temperature, :math:`T_0`. Given this simple relationship, the user should
+treat the temperature from this EOS as only a rough estimate.
+
+The Grunesien parameter is given by
+
+.. math::
+
+    \Gamma(\rho) =
+      \begin{cases}
+        \Gamma_0                                          & \rho < \rho_0 \\
+        \Gamma_0 \frac{\rho_0}{\rho} 
+           + b(1 - \frac{\rho_0}{\rho})                   & \rho >= \rho_0
+      \end{cases}
+
+and when the unitless user parameter :math:`b=0`, this ensures the the Gruneisen
+parameter is of a form where :math:`\rho\Gamma =` constant in compression.
+
+The reference pressure along the Hugoniot is determined by
+
+.. math::
+
+    P_H(\rho) = \rho_0 c_0^2 \mu
+      \begin{cases}
+        1                                                 & \rho < \rho_0 \\
+        \frac{1 + \left(1 - \frac{1}{2}\Gamma_0 \right)\mu - \frac{b}{2} \mu^2}
+          {\left(1 - (s_1 - 1)\mu + s_2 \frac{\mu^2}{1 + \mu}
+            - s_3 \frac{\mu^3}{(1+\mu)^2} \right)^2}      & \rho > \rho_0
+      \end{cases}
+
+where :math:`c_0`, :math:`s_1`, :math:`s_2`, and :math:`s_3` are fitting
+paramters. The units of :math:`c_0` are velocity while the rest are unitless.
```

Comment:
Perhaps worth commenting that the exact fitting form is "model implementation dependent" and our fitting formula may not look like the formula in another code.

Reply:
Are you referring to the fact that this looks a lot like a polynomial fit to the Us-up curve?

Reply:
Basically yes. In other codes I've seen different formulae here with different numbers of "s" coefficients. It has to do with the order of the fit.

Reply:
Added a clarifying sentence